client/webserver/site/src/js/app.js

```diff
@@ -367,8 +367,8 @@ export default class Application {
       return
     }
     State.clearAllStore()
-    State.removeCookies(authCK, darkModeCK)
-    window.location.reload()
+    State.removeAuthCK()
+    window.location.href = '/login'
```

Comment:
I'm not certain this is our preferred method of redirect, so I'll let @buck54321 comment.

Reply:
This is good.